controllers/v1/entityAssessorsController.js

```diff
@@ -449,4 +449,77 @@ module.exports = class EntityAssessors extends Abstract {
     });
   }
 
+    /**
+  * @api {post} /assessment/api/v1/entityAssessors/create/:programId?solutionId=solutionId Create entity assessors
+  * @apiVersion 1.0.0
+  * @apiName Create entity assessors
+  * @apiGroup Entity Assessor
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
```

Comment:
@aman-tunerlabs - We should put a restriction of internal access token here and pass userId as a query parameter, createdBy, updatedBy can set to "SYSTEM"